src/EPR.SubsidiaryBulkUpload.Application/Services/SubsidiaryService.cs

```diff
@@ -87,8 +88,10 @@ public async Task<bool> GetSubsidiaryRelationshipAsync(int parentOrganisationId,
 
             if (problemDetails != null)
             {
-                throw new ProblemResponseException(problemDetails, response.StatusCode);
+                _logger.LogError("Error occurred in GetCompanyByCompaniesHouseNumber call: {Parent} Subsidiary: {Subsidiary}", problemDetails.Detail, response.StatusCode);
```

Comment:
Check what you are logging in all of these methods - this code puts the problem detail into the {Parent} placeholder and the status code as {Subsidiary}.

Reply:
fixed.